.github/workflows/standard-deploy.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  workflow_call:
+    inputs:
+      app_name:
+        required: false
+        type: string
+      environment:
+        required: true
+        type: string
+      version:
+        required: true
+        type: string
+      db_name:
+        required: false
+        type: string
+        default: ''
+    secrets: 
+      AWS_ACCOUNT:
+        required: true
+
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Do we want this? If so we need some inputs

Reply:
Good point.  Removing...